### PR TITLE
[T5A1][T15-A3]Kenny Ng

### PR DESCRIPTION
**Pull request by kennyngdsc**
Work in progress

### CODE REVIEW

**Comment by kennyngdsc** [comment]
Ready for review

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | edit a person | change an old detail to a new one
```
Comment (sooyj) [inline doc/DeveloperGuide.md:48]: Pretty good user stories. Note that you may want to consider the priority level carefully :-).

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -73,13 +75,33 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Renaming of an existing tag
+
+**Actor**
+admin
```
Comment (sooyj) [inline doc/DeveloperGuide.md:82]: As discussed in tutorial. This probably should be user.

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -73,13 +75,33 @@ Use case ends.
...
+
+**MSS**
+
+1. Admin requests to list persons
```
Comment (sooyj) [inline doc/DeveloperGuide.md:86]: The steps could be better as listing persons seems like a wasted step.

File: doc/DeveloperGuide.md [hunk 4]
```diff
@@ -73,13 +75,33 @@ Use case ends.
...
+2. AddressBook shows a list of persons
+3. Admin decides to rename tag "friends" to "buddies"
+4. Admin edits all persons with "friends" to "buddies"
+Use case ends.
+
+**Extensions**
+
+2a. Admin is happy with all the conventions
```
Comment (sooyj) [inline doc/DeveloperGuide.md:94]: Try to be more precise :-) and focus on what the user do.

File: doc/DeveloperGuide.md [hunk 5]
```diff
@@ -73,13 +75,33 @@ Use case ends.
...
+
+4a. AddressBook shows an error message, "edit" is an invalid command
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be able to show how a filled AddressBook looks like upon "list" command via screenshot image.
```
Comment (sooyj) [inline doc/DeveloperGuide.md:104]: As discussed, this is probably something outside of the system.

**Review by sooyj — COMMENTED** [review]
A couple of issues mentioned in comment.

**Pull request closed without merging**